scripts/mir-perf.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -ex
+
+# find script's path
+pushd `dirname $0`
+DIR=`pwd`
+popd
+
+source $DIR/init.sh
+
+function on_exit {
+  set +e
+  source $DIR/cleanup.sh
+}
+
+trap on_exit EXIT
+
+if [ $# -lt 3 ]; then
+echo -e "${ERR} syntax: mir-perf <XenServer address> <XenServer root password> <library> <Run time>"
+exit
+fi
+
+XS=$1
+PASSWORD=$2
+LIBRARY=$3
+TRAFTIME=$4
+
+XENSERVER="sshpass -p $PASSWORD ssh -oStrictHostKeyChecking=no -l root $XS"
+
+pushd $DIR/../$LIBRARY
+
+eval `opam config env`
+env DHCP=true NET=direct mirage config --xen
+
+make
+
+popd
+
+# bug: XenServer returns: Ssl.Write_error(5)
+#$TMP/xe-unikernel-upload/xe-unikernel-upload \
+#    --username=root --password=$PASSWORD --uri=https://$XENSERVER/ \
+#    --path=$TMP/mirage-skeleton/iperf_lite/mir-stackv4.xen
+
+$XENSERVER 'if [[ ! -d /boot/guest ]]; then mkdir /boot/guest; fi'
+
```

Comment:
not familiar with that usage -- what does `[[` do, cf `[`?
(could also write this shorter `[ ! -d /boot/guest ] && mkdir /boot/guest` i guess too.)